### ISSUE TITLE
Ripple renames the nugets when downloading

### ISSUE THREAD

**Issue by andreasohlund** [FubuMvcArchive/ripple#200]
We have a nuget called: 

NServiceBus.4.1.0-Unstable7 

and when ripple downloads it (and updates nuspecs) it gets renamed to:

NServiceBus.4.1.0.0-Unstable7

From the log:

```
Downloading http://builds.particular.net/guestAuth/repository/download/NServiceBusCore_Build/24278:id/NServiceBus.4.1.0-
Unstable7.nupkg to C:\Users\andreas.ohlund\AppData\Roaming\ripple\NServiceBus.4.1.0.0-Unstable7.nupkg

```

Is there a reason for this?

**Comment by jmarnold** [comment]
When we're dealing with the floating feeds, we parse the nuget information
from the file name. In this case, it looks like:

Name: NServiceBus
Version: 4.1.0-Unstable7

When we go to reconstruct the name, the version is an instance of
SemanticVersion. So we're dealing with a canonical version vs. the raw
string at that point. We could try to make it smarter so that it maintained
the original name if it's a huge deal.

**Comment by andreasohlund** [comment]
No big deal I guess, just got confused!

What about the nuspec autoupdates?(will the original version be used there?)

**Comment by jmarnold** [comment]
The version may vary slightly but only due to a leading zero. Just depends
on which token you use -- remember, we support NextMinor, NextMajor,
Current. So you have control over the granularity of the version ranges.

**Comment by andreasohlund** [comment]
Cool'

Closes #200
